styles.css

```diff
@@ -0,0 +1,1240 @@
+* {
+    box-sizing: border-box;
+    margin: 0%;
+    padding: 0%;
+
+}
+
+#layout {
+    height: 100vh;
+    display: grid;
+    grid-template:
+
+        "side    left-header   left-header     main-header     right-header   side2"50px "side    list-logo     list-logo    middle-main     main-right     side2"15% "side    list-logo    list-logo    main            main-right     side2"1fr "side     tweak         tweak    main              main-right    side2"75px "side     bottom-left  bottom-left     main            bottom-right  side2"70px/ 1fr 0.8fr 1.3fr 6fr 3fr 0.8fr;
+
+}
+
+.side {
+    grid-area: side;
+    background-color: white;
+    ;
+}
+
+.left-header {
+    grid-area: left-header;
+    margin-right: 70%;
+    margin-left: 10%;
+    margin-top: 5%;
+
+}
+
+.left-header img {
+    object-fit: contain;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+.list-logo {
+    grid-area: list-logo;
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    margin-top: 4mm;
+    margin-left: 4mm;
```

Comment:
It's not really recommended to use `mm` as a unit for creating webpages. It's more useful when creating styles that are specifically designed to be printed as that way the real world measurements relate to the paper being printed.